roles/maintenance_33_varnish/tasks/main.yml

```
@@ -0,0 +1,82 @@
+---
+- name: This task only serves as a template for the tasks below
+  ansible.builtin.set_fact:
+    ignoreme: &task
+      name: "{{ vars.taskid }}: {{ vars.name }}"
+      register: task
+      when:
+        - "vars.taskid not in maintenance_exclude_tasks"
+        - "maintenance_only is not defined or maintenance_only == vars.taskid"
+  vars:
+    taskid: ignoreme
+    name: bar
+
+- <<: *task
+  vars:
+    taskid: 33-011
+    name: "Check which varnish configuration is active"
+  ansible.builtin.shell:
+    cmd: "systemctl show -p ExecStart varnish | grep 'ExecStart' | grep -oE '\\-f\\s*[a-z\\/\\.A-Z0-9]+' | grep -oE '[a-z\\/\\.A-Z0-9]{2,}'"
+  register: varnish_active_configfile
+  ignore_errors: true
+  changed_when: false
+- <<: *task
+  vars:
+    taskid: 33-011
+    name: "Check which varnish configuration is active"
+  ansible.builtin.stat:
+    path: "{{ varnish_active_configfile.stdout }}"
+  register: varnish_active_configfile_stat
+  ignore_errors: true
+  changed_when: not varnish_active_configfile_stat.stat.exists
+
+
+- <<: *task
+  vars:
+    taskid: 33-012
+    name: "Config: Are there any errors in this configuration?"
+  ansible.builtin.shell:
+    cmd: "varnishd -Cf {{ varnish_active_configfile.stdout }} > /dev/null 2>&1"
```

Is there a reason for the redirect to /dev/null? I know it's in the original checklist for manual execution, but I don't think we need it in the Ansible implementation:

```suggestion
  ansible.builtin.command: "varnishd -Cf {{ varnish_active_configfile.stdout }}"
```

---

It prints the absolute entire compiled varnish config. So for the case where you want to have the output of the ok tasks to be printed, this fills your terminal and when not, I don't see the harm in using it.